### ISSUE TITLE
Custom colormap(s) for plot.flow()

### ISSUE THREAD

**Issue by jondo** [adalca/neuron#60]
... would be nice to have, but are not implemented yet, see https://github.com/adalca/neurite/blob/249d4978c706fc21c87e192bb11a4d20169cafc5/neurite/py/plot.py#L297 
(Sorry, I can't take the time to offer a patch.)

**Comment by adalca** [comment]
@jondo agreed, but also limited by time :). I'll keep this open in case someone finds the time to help us out!
